pgjdbc/src/test/java/org/postgresql/test/jdbc3/CompositeTest.java

```diff
@@ -76,6 +79,18 @@ void simpleSelect() throws SQLException {
     assertEquals("(1,2.2,)", pgo.getValue());
   }
 
+  @Test
+  void testAddType() throws SQLException {
+    PGConnection pgconn = conn.unwrap(PGConnection.class);
+    pgconn.addDataType("\"Composites\".\"SimpleComposite\"", SimpleComposite.class);
+    PreparedStatement pstmt = conn.prepareStatement("SELECT '(1,2.2,a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11)'::\"Composites\".\"SimpleComposite\"");
```

Comment:
Could you please clarify how does this test differ from `org.postgresql.test.jdbc2.CustomTypeWithBinaryTransferTest#testCustomBinaryTypes` ?